draft-ietf-tls-tls13.md

```diff
@@ -3708,13 +3686,9 @@ the TLS PRF. This document replaces the PRF with HKDF, thus requiring
 a new construction. The exporter interface remains the same, however
 the value is computed as:
 
-    HKDF-Expand-Label(HKDF-Extract(0, exporter_secret),
+    HKDF-Expand-Label(exporter_secret
```

Comment:
missing comma after first argument, here

(noted previously, but comment appears to have gotten lost on GitHub for some reason)